examples/rbd/storageclass.yaml

```diff
@@ -34,20 +25,19 @@ parameters:
    imageFeatures: layering
 
    # The secrets have to contain Ceph admin credentials.
-   # NOTE: If using "clusterID" instead of "monitors" above, the following
-   # secrets MAY be added to the ceph-cluster-<cluster-id> secret and skipped
-   # here
+   # NOTE: The following secrets MAY be added to the ceph-cluster-<cluster-id>
+   # secret and skipped here
    csi.storage.k8s.io/provisioner-secret-name: csi-rbd-secret
    csi.storage.k8s.io/provisioner-secret-namespace: default
    csi.storage.k8s.io/node-publish-secret-name: csi-rbd-secret
    csi.storage.k8s.io/node-publish-secret-namespace: default
 
    # Ceph users for operating RBD
-   # NOTE: If using "clusterID" instead of "monitors" above, the following
-   # IDs MAY be added to the ceph-cluster-<cluster-id> secret and skipped
-   # here
+   # NOTE: If the secrets above are skipped, then the following values are
```

Comment:
Shouldn't it pull the adminid and userid from the secret? Why are there multiple places to store this data?

Reply:
Resolved the text, this was left over comments when the secret was removed.